src/app/pages/CreateConnectorPage/StepReview.tsx

```diff
@@ -161,91 +160,92 @@ export const StepReviewComponent: React.FC<StepReviewComponentProps> = ({
-        )}
-        {toggleView ? (
-          <ViewJSONFormat
-            kafka={kafka}
-            namespace={namespace}
-            connectorType={connectorType}
-            name={name}
-            topic={topic}
-            userErrorHandler={userErrorHandler}
-            userServiceAccount={userServiceAccount}
-            configString={configString}
-            configurationSteps={configurationSteps}
+      {savingError && (
+        <FormAlert>
+          <Alert
+            variant="danger"
+            title={savingError}
+            aria-live="polite"
+            isInline
           />
-        ) : (
-          <>
-            <Grid>
-              <GridItem span={4}>
-                <strong>{t('connectorCategory')}</strong>
-              </GridItem>
-              <GridItem span={8}>
+        </FormAlert>
+      )}
+      {toggleView ? (
+        <ViewJSONFormat
+          kafka={kafka}
+          namespace={namespace}
+          connectorType={connectorType}
+          name={name}
+          topic={topic}
+          userErrorHandler={userErrorHandler}
+          userServiceAccount={userServiceAccount}
+          configString={configString}
+          configurationSteps={configurationSteps}
+        />
+      ) : (
+        <>
+          <dl className="pf-c-description-list pf-m-horizontal">
+            <div className="pf-c-description-list__group">
+              <dt className="pf-c-description-list__term">
+                {t('connectorCategory')}
+              </dt>
+              <dd className="pf-c-description-list__description">
                 {(connectorType as ConnectorTypeAllOf).description}
-              </GridItem>
-            </Grid>
-            <Grid>
-              <GridItem span={4}>
-                <strong>{t('kafkaInstance')}</strong>
-              </GridItem>
-              <GridItem span={8}>{kafka.name}</GridItem>
-            </Grid>
-            <Grid>
-              <GridItem span={4}>
-                <strong>{t('namespace')}</strong>
-              </GridItem>
-              <GridItem span={8}>{namespace.name}</GridItem>
-            </Grid>
-
-            <Title headingLevel="h3" size={TitleSizes['2xl']}>
+              </dd>
+            </div>
+            <div className="pf-c-description-list__group">
+              <dt className="pf-c-description-list__term">
+                {t('kafkaInstance')}
+              </dt>
+              <dd className="pf-c-description-list__description">
+                {kafka.name}
+              </dd>
+            </div>
+            <div className="pf-c-description-list__group">
+              <dt className="pf-c-description-list__term">{t('namespace')}</dt>
+              <dd className="pf-c-description-list__description">
+                {namespace.name}
+              </dd>
+            </div>
+            <Title
+              headingLevel="h3"
+              size={TitleSizes['xl']}
+              className="--pf-c-title--Size"
+            >
```

Comment:
For the `<Title>` components that display after the primary Review title on this page, we want them to be visually and semantically one step lower/smaller in the hierarchy to convey that relationship. 

I think the Review title is implemented with https://github.com/bf2fc6cc711aee1a0c2a/cos-ui/blob/b2bfa3c29dd700636b2825e4523506d3a806bf6a/src/app/components/StepBodyLayout/StepBodyLayout.tsx#L25 which is using the `<h2>` heading level, and defaults to the `xl` size.  That means the `h3` level for this "Basic" title is good, but we probably want to shrink the size by one to `lg` instead of `xl` like it is here.

Reply:
BTW, you can see what the default sizes would be with this example: https://patternfly-react-main.surge.sh/components/title#default-sizes